src/sections/Topics/Topic-sidebar/index.js

```diff
@@ -0,0 +1,114 @@
+                ){
+                    group(field: frontmatter___tags) {
+                        fieldValue
+                        totalCount
+                    }
+                }
+                categories: allMdx(
+                    filter: { fields: { collection: { eq: "topics" } }, frontmatter: { published: { eq: true } } }
+                ){
+                    group(field: frontmatter___category) {
+                        fieldValue
+                        totalCount
+                    }
+                }
+            }
+        `
+  );
+
+  const tags = data.tags.group.sort((a,b) => {
+    return b.totalCount - a.totalCount;
+  });
+  const categories = data.categories.group.sort((a,b) => {
+    return b.totalCount - a.totalCount;
+  });
+
+  return (
+    <TopicSideBarWrapper>
+      {/*<div className="sidebar-widgets">*/}
+      {/*  <div className="search-box">*/}
+      {/*    <input type="text" placeholder="Search here..." />*/}
+      {/*    <Button>*/}
+      {/*      <FaSearch />*/}
+      {/*    </Button>*/}
+      {/*  </div>*/}
+      {/*</div>*/}
+      <div className="explain-1">
+        <div className="cards">
+          <a href="https://discuss.layer5.io/">
+            <div className="card">
+              <h2>Join the Conversation</h2>
+              <p>On our Discussion Forum</p>
+              <img className="logo" src={Discuss} />
+            </div>
+          </a>
+        </div>
+      </div>
+
+      <div className="sidebar-widgets catagorie">
+        <div className="widgets-title">
+          <h3>Categories</h3>
+        </div>
+        <ul>
+          { categories && categories.map(category => (
+            <li key={category.fieldValue}>
+              <Link to={`/topics/category/${slugify(category.fieldValue)}`}>
+                <span>{category.fieldValue}</span>
+                <em>({category.totalCount})</em>
+              </Link>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="subscribe">
+        <form name="contactform" method="post" action="https://calcotestudios.us15.list-manage.com/subscribe/post?u=6b50be5aea3dfe1fd4c041d80&amp;id=6bb65defeb">
+          <div>
+            <span>
+              <h4>Subscribe to our Newsletter</h4>
+            </span>
+            <input className="inputrow subscribe-email" type="email" placeholder="Email Address" name="EMAIL" id="mce-EMAIL" required />
+            <Button secondary title="Subscribe" id="mc-embedded-subscribe" />
+          </div>
+        </form>
+      </div>
+      
+      <div className="sidebar-widgets tags">
+        <div className="widgets-title">
+          <h3>Tags</h3>
+        </div>
+        <ul>
+          { tags && tags.map(tag => (
+            <li key={tag.fieldValue}>
+              <Link to={`/topics/tag/${slugify(tag.fieldValue)}`}>{tag.fieldValue} ({tag.totalCount})</Link>
```

Comment:
```suggestion
              <Link to={`/topic/tag/${slugify(tag.fieldValue)}`}>{tag.fieldValue} ({tag.totalCount})</Link>
```
@leecalcote the tags seem to be working with https://617cc9696961c5134972bc6a--layer5ng.netlify.app/topic/tag/envoy this URL 🤔

Reply:
Aha! Thanks for figuring this out, ⚡️@debo19